ui/src/app/cohort-search/cohort-search/cohort-search.component.spec.tsx

```diff
@@ -0,0 +1,19 @@
+import {shallow} from 'enzyme';
+import * as React from 'react';
+
+import {currentWorkspaceStore, queryParamsStore} from 'app/utils/navigation';
+import {workspaceDataStub} from 'testing/stubs/workspaces-api-stub';
+import {CohortSearch} from './cohort-search.component';
+
+describe('CohortSearch', () => {
+  beforeEach(() => {
+    currentWorkspaceStore.next(workspaceDataStub);
+    queryParamsStore.next({
+      cohortId: 'test-id'
+    });
+  });
+  it('should render', () => {
+    const wrapper = shallow(<CohortSearch setCohortChanged={() => {}} setShowWarningModal={() => {}} setUpdatingCohort={() => {}}/>);
+    expect(wrapper).toBeTruthy();
+  });
+});
```

Comment:
Can we add more test like for search request with include/exclude?

Reply:
Probably could use more still, but I added one to check that the number of groups matches the length of the includes/excludes array.